packages/suite/src/components/wallet/CoinControl/UtxoSelection.tsx

```diff
@@ -123,14 +125,21 @@ interface UtxoSelectionProps {
 }
 
 export const UtxoSelection = ({ isChecked, transaction, utxo }: UtxoSelectionProps) => {
+    const { account, network, selectedUtxos, toggleUtxoSelection } = useSendFormContext();
+
     const device = useSelector(state => state.suite.device);
+    const coinjoinAccount = useSelector(state => selectCoinjoinAccountByKey(state, account.key));
     const { openModal } = useActions({
         openModal: modalActions.openModal,
     });
 
-    const { account, network, selectedUtxos, toggleUtxoSelection } = useSendFormContext();
     const theme = useTheme();
 
+    const isRegisteredForCoinjoin = coinjoinAccount?.session?.registeredUtxos.includes(
+        getUtxoOutpoint(utxo),
+    );
+    const isUnavailableForCoinjoin =
+        account.accountType === 'coinjoin' && Number(utxo.amount) < 5000; // TODO: should include UTXOs banned from CoinJoin as well
```

Comment:
maybe BigNumber for utxo.amount? Why 5000? I don't understand this condition.

Reply:
Thanks, 6433f762959dbbd57722c197fdbdb73e7c013b61 explains this. It is WIP.